package.json

```diff
@@ -33,13 +33,17 @@
 		"string"
 	],
 	"dependencies": {
-		"normalize-url": "^7.0.2",
-		"re2": "^1.17.2",
```

Comment:
?

Reply:
`re2` isn't used anywhere in the project

Reply:
It's required by `url-regex-safe`. Usually, `git blame` is a good way to find out why something exists.

Reply:
Makes sense, will add it back. Requiring a build step for this library feels a bit too much - I might open up an issue for discussion on using an alternate implementation

Reply:
Yeah, we can switch to https://github.com/sindresorhus/super-regex instead. PR welcome.